sourced/ml/cmd_entries/merge_df.py

```diff
@@ -0,0 +1,8 @@
+from sourced.ml.models import MergeDocFreq
+
+
+def merge_df_entry(args):
+    merger = MergeDocFreq(ordered=args.ordered,
+                          vocabulary_size=args.vocabulary_size,
+                          min_docfreq=args.min_docfreq)
+    merger.convert(args.input, args.output, pattern=args.filter)
```

Comment:
This class will be simplified too - you will feed only the list of input files and the output name. The compilation of that list is outside of the class.